lib/validate.test.js

```diff
@@ -166,5 +240,69 @@ describe(__filename, function () {
           done();
         });
     });
+
+    it('should strip the where parameter and accept the list request', function (done) {
+      request
+        .get('/gadgets?who=Sue&where=there')
+        .expect(200)
+        .expect('content-type', /json/)
+        .end(function (err, res) {
+          expect(err).to.not.exist;
```

Comment:
It looks like we don't validate the params are stripped. I think perhaps the easiest way to do so is to pass either req.body or req.query to a sinon stub/spy on line 59/81, and check that here in this block. Thoughts?